### ISSUE TITLE
Issue with k8s.io/docs/concepts/services-networking/dns-pod-service/

### ISSUE THREAD

**Issue by gaurav-mittal-helm** [kubernetes/website#19345]
**This is a Bug Report**

**Problem:**
We should also capture a section which talks about how to set up the additional DNS Server along with the default one that is managed by k8s cluster.

**Proposed Solution:**

**Page to Update:**
https://kubernetes.io/...

1.14

**Comment by Colstuwjx** [comment]
Just to eliminate your confusion, the [Pod’s DNS Config](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-config) section did talk how to customize pod dns config, just set `dnsPolicy` to `None`, and `nameservers` to your own DNS servers.

/triage support
/language en

**Comment by gaurav-mittal-helm** [comment]
Hi,

If we set the dnsPolicy to None, then the default k8s DNS will no more be applicable. Then what will be the impact on the pod communication within the cluster wrt other pods when the name is used from communication.

My use case is keeping the default behavior of the k8s cluster communication intact, what to add an extra DNS entry so that some FQDN that are resolvable from external DNS can also be done.

I tried by adding manually the entry in /etc/resolv.conf inside the pod it worked. But how to do it at k8s level.

With Warm Regards,
Gaurav Mittal
